example/dto/example1/Test1Response.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema in the source OAS file,
+ * and use `yarn brij dto` to regenerate this file.
+ */
+
+import { JSONSchema } from '@kong/brij'
+
+export interface SharedObj1 {
+  [k: string]: unknown
+}
+
+export type Test1Response = SharedObj1
```

Comment:
This is the line that is added by the new code, it ensures that we export a type `Test1Response` that is the same name as the instance exported on line 25 below. This way, when importing, you can use a single name for both the type and the validator instance, e.g., `import { Test1Response } from '../Test1Response.ts` will give you both the type and instance.